superset/views/base.py

```diff
@@ -333,6 +333,7 @@ def cached_common_bootstrap_data(  # pylint: disable=unused-argument
         "d3_format": conf.get("D3_FORMAT"),
         "d3_time_format": conf.get("D3_TIME_FORMAT"),
         "currencies": conf.get("CURRENCIES"),
+        "deckgl_tiles": conf.get("DECKGL_BASE_MAP"),
```

Comment:
Not sure if it should be in scope for this PR, but we're looking to avoid growing the `bootstrap_data` payload, as it's sent to the frontend globally for all page loads. Since the set of defaults ones are hard-coded on the frontend, maybe we make the default in `superset/config.py` empty, and only pay the price of growing the payload in environments looking to customize the set (?)

Reply:
Hi,
Indeed It's a good idea, I set the  DECKGL_BASE_MAP default to None